### ISSUE TITLE
Possible fix for outdated mixins problems

### ISSUE THREAD

**Issue by dark-penguin** [DimensionalDevelopment/VanillaFix#265]
There seems to be a lot of problems with outdated mixins. Apparently, one solution is to make sure VanillaFix is always loaded first. This could be done by renaming the jar file, but it's impossible for those who use the Twitch launcher.

Seeing that VanillaFix seems to be up to speed with the latest mixins, and due to the nature of this mod, wouldn't it be beneficial to rename the official jar to "_VanillaFix", for example? Or maybe specify in some other way that it should always be loaded before all other mods. Of course, this is not really VanillaFix's fault to fix, but this could probably make life easier for a lot of people.

**Comment by GiggleMaster** [comment]
Agreed. I just tried this fix by renaming it do _VanillaFix and it works again now. Thanks, man!
I hope the mixins get fixed. If not I hope that the file itself gets renamed to be future proof.

**Comment by dark-penguin** [comment]
As far as I understand, the problem is that VanillaFix does update to the latest version of mixins, but other mods do not, and this causes the crash (unless VanillaFix runs first and apparently initializes something properly...). Since we can not force all mod creators to update their mods, we could try to mend the situation to a certain degree this way.

**Comment by MechanosG** [comment]
@dark-penguin Hey, thanks for the rename workaround!  I just installed Performant and immediately started getting the infamous mixin error.  I went around in circles with this error with lag goggles once, and eventually just gave up and stayed with an old version of lag goggles.  Anyway, I saw a mention of VanillaFix over on the Performant issue page, so I figured I'd check the issues here.  Saw this thread right away, tried renaming VanillaFix to _VanillaFix, and it worked first try.

...I wonder if I could have done something similar with lag goggles months ago.  -facepalm-

**Comment by wootaboot** [comment]
Worked like a charm, thanks

**Comment by DRAKKKS** [comment]
mine still crashes with malisis core

**Comment by Lucas559-noob** [comment]
Im using multimc, I renamed the mod but still get the error, a way to fix it?
